### DISCUSSION TITLE
Vitrus total makr the release .exe file as virus

### DISCUSSION THREAD

**Discussion by Arwagollon** [HarukaMa/starfield-xgp-import#8] [General]
Probably false positive, but I think you should know about it.   Vitus total score (6/70)
link: https://www.virustotal.com/gui/file/0adb8aa4e6c6a5c5548e0572213d35fe2dfefcb34f5a7f358dcfd23cc5154689

**Comment by HarukaMa** [comment]
Guess they don't like pyinstaller. XGP save extractor has those reports as well. 

I might try another packaging tool if there will be a next version.
